### ISSUE TITLE
Tried to initialize from a previous compilation, but couldn't.

### ISSUE THREAD

**Issue by IrshadS** [dart-lang/sdk#36336]
Thank you for taking the time to file an issue!

In order to route, prioritize, and act on this, please include:

* Dart SDK Version (`dart --version`)
* Whether you are using Windows, MacOSX, or Linux (if applicable)
* Whether you are using Chrome, Safari, Firefox, Edge (if applicable)

Missing some or all of the above might make the issue take longer or be
impossible to act on.

----

Is it really an issue? For general questions consider starting with Stack
Overflow:
https://stackoverflow.com/questions/tagged/dart

Also consider our Gitter channel for light-weight/quick discussions:
https://gitter.im/dart-lang/sdk

-----

**Comment by mraleph** [comment]
@jensjoha @aam 

Maybe we should either a) include a lot of actionable information into the error report for this error or b) suppress this error entirely?

**Comment by jensjoha** [comment]
There actually is a lot of actionable information in the warning these days. It contains both a stacktrace and asks the user to upload the dill file it tries to initialize from.

In my opinion we should not suppress it --- it was added because we got into a situation that in flutter - without anyone noticing - loading from a dill file didn't work at all.
The hope is that by issuing warnings when it fails (except for in a few cases where it is "ok") we'll stop that from happening again.
